docker-compose.yml

```diff
@@ -62,6 +62,12 @@ x-ccache: &ccache
   CCACHE_MAXSIZE: 1G
   CCACHE_DIR: /ccache
 
+x-sccache: &sccache
+  AWS_ACCESS_KEY_ID:
+  AWS_SECRET_ACCESS_KEY:
+  SCCACHE_BUCKET:
```

Comment:
Does it mean they are inherited from the calling process environment?

Reply:
Yes, saving us the work to add the mall manually via `-e`